CHANGELOG.md

```diff
@@ -29,6 +29,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - Import maps now apply to modules in external dependencies, not just to local
   project files.
 
+- The TypeScript compiler can now deal with multiple versions of the same
```

Comment:
What makes a change breaking? Could some subtle import or more nuanced behavior in the import resolution make this a breaking change?